### PR TITLE
fix Ingresses version

### PR DESCRIPTION
**Pull request by TheoBrigitte**
see [`#inc-2021-09-03-analytics-panel-app-is-in-unknown-error-state`](https://gigantic.slack.com/archives/C02E11976M6/p1630664492000300)

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
Using capabilities looks more correct, e.g: https://github.com/giantswarm/api/blob/master/helm/api/templates/ingress.yaml#L1

**Comment by mig4** [comment]
@TheoBrigitte, do you want to try this trick Joe mentioned: https://github.com/giantswarm/api/blob/master/helm/api/templates/ingress.yaml#L1 ?

**Comment by JosephSalisbury** [comment]
theo i really think we should use the trick from the api

**Comment by JosephSalisbury** [comment]
lolololol

**Comment by mig4** [comment]
You'll also need this: https://github.com/giantswarm/api/blob/master/helm/api/templates/ingress.yaml#L36

**Review by JosephSalisbury — APPROVED** [review]
lgtmiiwfy

**Comment by TheoBrigitte** [comment]
Tested on ginger :heavy_check_mark:

**Review by JosephSalisbury — APPROVED** [review]

File: helm/macropower-analytics-panel-server/templates/ingress-auth.yaml [hunk 1]
```diff
@@ -14,9 +18,14 @@ spec:
     http:
       paths:
       - backend:
+{{ if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}
           service:
-            name: grafana
+            name: {{ include "resource.default.name" . }}
```
Comment (mig4) [inline helm/macropower-analytics-panel-server/templates/ingress-auth.yaml:23]: This should actually be `grafana` IIRC

**Review by mig4 — CHANGES_REQUESTED** [review]

File: helm/macropower-analytics-panel-server/templates/ingress-auth.yaml (hunk 1 above)
Comment (TheoBrigitte) [reply helm/macropower-analytics-panel-server/templates/ingress-auth.yaml]: good catch

**Review by mig4 — APPROVED** [review]

**Pull request merged**